### ISSUE TITLE
Add a form field in the profile section that allows users to customize the initials displayed in their profile avatars.

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#10489]
**Describe the problem you would like to solve**
Using initials as avatars adds a personal touch to user profiles, giving them a unique identity without the need for personal photos. This also addresses privacy concerns related to using images. However, some users may prefer to adjust the automatically generated initials to better suit their preferences.

**Describe the solution you'd like**
Add a form field under **User Profile > Identity** that lets users customize their initials. They should be able to enter only 2 characters, no more.

**Comment by taslangraham** [comment]
I've added a PR for this @ewhanson. Please take a look.
**pkp-lib** - https://github.com/pkp/pkp-lib/pull/10639

**submodule updates**
 ojs - https://github.com/pkp/ojs/pull/4533
ops - https://github.com/pkp/ops/pull/818
omp - https://github.com/pkp/omp/pull/1776


There's an avatar [component ](https://github.com/pkp/ui-library/tree/main/src/components/UserAvatar)in the UI library currently used to display initials based on the user's name. This would need to be updated, in a separate issue,  to accept custom initials

**Comment by ewhanson** [comment]
Looks good, @taslangraham. Just a few questions in pkp-lib. Once this is merged, feel free to create the issue and update the avatar component as well.

**Comment by taslangraham** [comment]
@ewhanson Thanks for the review

**Comment by taslangraham** [comment]
@ewhanson I've made small updates to pkp-lib PR based on your initial feedback. Please take a look.

**Comment by ewhanson** [comment]
Thanks, @taslangraham! Just one comment, but once you've addressed that, feel free to merge.
